src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -134,6 +145,7 @@ private void ConfigureLanguageServer(RazorLanguageServerBuilder builder)
             var services = builder.Services;
             var loggerProvider = (FeedbackFileLoggerProvider)_feedbackFileLoggerProviderFactory.GetOrCreate();
             services.AddSingleton<ILoggerProvider>(loggerProvider);
+            services.AddSingleton<LanguageServerFeatureOptions, VSLanguageServerFeatureOptions>(s => _vsLanguageServerFeatureOptions);
```

Comment:
This doesn't control precedence, and should actually be 
```suggestion
            services.AddSingleton<LanguageServerFeatureOptions>(_vsLanguageServerFeatureOptions);
```

is there a way to force use of this `LanguageServerFeatureOptions` over the default?